UNRELEASED_CHANGELOG.md

```diff
@@ -39,8 +39,43 @@
 ### ⬆️ Improved
 
 ### ✅ Added
+- Added the following use case functions to `ChatDomain` which are supposed to replace `ChatDomain.useCases` property:
+* `ChatDomain::replayEventsForActiveChannels`
```

Comment:
Could you add an small description of what every method do?

Reply:
Added